src/follower.py

```diff
@@ -66,11 +66,18 @@ def __init__(self):
         self._tag_lost_time = 0
         self._speed_cycle_time = time()
 
+        self.camera_angle_offset = 0
```

Comment:
Why isn't this a private member? No outside class should have access to the panning angle.

Reply:
I believe there was a request above to remove the privatization then go back and put them in, but I can do that now.